modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase

```diff
@@ -133,14 +133,74 @@ definition {
 		}
 	}
 
-	@description = "[Please Update]"
-	@ignore = "Test Stub"
+	@description = "Create multiple orders and use the search bar to verify if it works correctly"
```

Comment:
This should be something like "Verify orders can be search when importing an order"